src/tvp/feature.ts

```diff
@@ -0,0 +1,142 @@
+import { BaseLanguageClient, DynamicFeature, RegistrationData, Event } from 'coc.nvim'
+import {
+  ClientCapabilities, RPCMessageType, ServerCapabilities, Emitter,
+  ExecuteCommandParams, ExecuteCommandRequest, TextDocumentPositionParams, TextDocument, Position,
+  NotificationType, RequestType, Disposable
+} from 'vscode-languageserver-protocol'
+import {
+  TreeViewChildrenParams, TreeViewChildrenResult, TreeViewDidChangeParams,
+  TreeViewNodeCollapseDidChangeParams, TreeViewVisibilityDidChangeParams, TreeViewNode,
+  MetalsTreeRevealResult
+} from './domain'
+import { commands } from 'coc.nvim'
+import { TreeViewProvider } from './provider'
+
+interface TreeViewParams { }
```

Comment:
Are we able to complete these interfaces?

Reply:
Removed